packages/vm/src/evm/types.ts

```diff
@@ -1,4 +1,21 @@
+import { OpHandler } from './opcodes'
+import { AsyncDynamicGasHandler, SyncDynamicGasHandler } from './opcodes/gas'
+
 /**
  * Log that the contract emitted.
  */
 export type Log = [address: Buffer, topics: Buffer[], data: Buffer]
+
+export type DeleteOpcode = {
+  opcode: number
+}
+
+export type AddOpcode = {
+  opcode: number
+  opcodeName: string
+  baseFee: number
+  gasFunction?: AsyncDynamicGasHandler | SyncDynamicGasHandler
+  logicFunction: OpHandler
+}
```

Comment:
Not a totally strong opinion here, but I wonder if it's enough if we publicly only expose the `CustomOpcode` type here? 🤔

Tendentially always a fan of keeping the exposed interface/API lean if possible.

Reply:
Ah wait this is actually the case. I have exported them specifically so you can get this extra typescript info. Otherwise it is not super clear how to use `DeleteOpcode`, if we export it you can use it without reading the manual :smile: 

But can still remove it if that is better, have no strong opinion

Reply:
Ok, let's keep, also no super strong opinion here